controllers/projects.controller.js

```diff
@@ -0,0 +1,17 @@
+const catchAsync = require('../utils/catchAsync')
+const ApiError = require('../utils/ApiError')
+const db = require('../database/connection')
```

Comment:
db module shall not be used inside controller use models/project. 
note d is not used.